ST_structs.h

```diff
@@ -1,333 +1,721 @@
-  RealF max_rate,      /* intrin_rate * proportion*/
+      /** \brief Max age of mature plant. Also used to determine annual species. */
+  IntUS max_age,
+      /** \brief Annuals: max years of viability of seeds. */
+        viable_yrs,
+      /** \brief Max seedlings that can establish in 1 year. */
+        max_seed_estab,
+      /** \brief Max vegetative regrowth units, for example tillers. */
+        max_vegunits,
+      /** \brief Years slow growth is allowed before mortality. */
+        max_slow,
+      /** \brief Index of this species in the Species variable.
+       * \sa Species */
+        sp_num,
+      /** \brief Resource group that this species belongs to. 
+       * \sa RGroup */
+        res_grp;
+      /** \brief Max growth rate. Defined as intrin_rate * proportion. */
+  RealF max_rate,
+      /** \brief Intrinsic growth rate. */
         intrin_rate,
+      /** \brief Starting size of a new individual of this species, relative to an adult plant. */
         relseedlingsize,
+      /** \brief Starting biomass in grams of a new individual of this species. */
         seedling_biomass,
-        mature_biomass,    /* biomass of mature_size individual */
-        seedling_estab_prob_old,  /* supports Extirpate() and Kill() */
+      /** \brief Biomass in grams of a mature individual. */
+        mature_biomass,
+      /** \brief Holds seedling establishment probability directly from inputs. 
+       * \sa seedling_estab_prob */
+        seedling_estab_prob_old,
+      /** \brief Seedling establishment probability.
+       * \sa seedling_estab_prob_old */
         seedling_estab_prob,
+      /** \brief Annual mortality probability. This variable is currently unused. */
         ann_mort_prob,
+      /** \brief Used in age-independent mortality.
+       * \sa _age_independent() */
         cohort_surv,
-        exp_decay,        /* annuals: exponent for viability decay function */
-        prob_veggrow[4],  /* 1 value for each mortality type, if clonal*/
-  	sd_Param1,	  /* for seed dispersal */
-  	sd_PPTdry,
-	sd_PPTwet,
-	sd_Pmin,
-	sd_Pmax,
-	sd_H,
-	sd_VT;
+      /** \brief Annuals-specific exponent for viability decay function. */
+        exp_decay,
+      /** \brief 1 value for each mortality type, if clonal*/
+        prob_veggrow[4],
+      /** \brief Seed dispersal parameter read from inputs.
+       * \sa ST_seedDispersal.c */
+  	    sd_Param1,
+      /** \brief Seed dispersal parameter read from inputs.
+       * \sa ST_seedDispersal.c */
+  	    sd_PPTdry,
+      /** \brief Seed dispersal parameter read from inputs.
+       * \sa ST_seedDispersal.c */
+	      sd_PPTwet,
+      /** \brief Seed dispersal parameter read from inputs.
+       * \sa ST_seedDispersal.c */
+	      sd_Pmin,
+      /** \brief Seed dispersal parameter read from inputs.
+       * \sa ST_seedDispersal.c */
+  	    sd_Pmax,
+      /** \brief Seed dispersal parameter read from inputs.
+       * \sa ST_seedDispersal.c */
+	      sd_H,
```

Comment:
average release height of the inflorescences in cm

Reply:
Addressed by commit 984bd13.